### DISCUSSION TITLE
OS Simulator not working in windows 10

### DISCUSSION THREAD

**Discussion by juanram0n** [zepp-health/zeppos-forum#66] [Q&A] [answered]
Thanks in advance.

I'm really interested in make apps for zepp os, but I tried a lot of things but simulator can't start, neither older versions.

Console throws this message:
`[firmware_simulator]: .\qemu_win\qemu-system-arm.exe: -net tap,ifname=tun: Device 'tap' could not be initialized`

I had downloaded GTR4 simulator.

Thanks a lot!!

**Comment by zdunex25** [comment, marked as answer]
You did not install Virtual network card properly: https://docs.zepp.com/docs/guides/tools/simulator/setup/#1-installing-a-virtual-network-card

**Reply by juanram0n** [reply]
Ups you are right!! thanks a lot I feel a little bit stupid right now  because don't follow the documentation 😅

**Reply by zdunex25** [reply]
We all've been through this, no worries :)
